src/Config/ConfigLocator.php

```diff
@@ -101,7 +104,7 @@ public function __construct($envPrefix = '')
         }
         $this->config->addPlaceholder(self::USER_CONTEXT);
         $this->config->addPlaceholder(self::DRUPAL_CONTEXT);
-        $this->config->addPlaceholder(self::SITE_CONTEXT); // not implemented yet (multisite)
```

Comment:
n.b. "not implemented yet (mulitsite)" comment.

Reply:
This was removed intentionally, as this PR adds site support.

Reply:
Yeah I made that note for @weitzman to point out that this PR is implementing a TODO. Then I forgot to save the comment that referenced it.

Reply:
Oh, it did get saved; see above.